libraries/cli/lib/bin.ts

```diff
@@ -55,7 +55,10 @@ async function main() {
   yargs = ModuleConfiguration.configureOptions(yargs);
 
   // Parse the arguments and config using only the base options
-  const baseArguments = yargs.wrap(yargs.terminalWidth()).parseSync(args);
+  const baseArguments = yargs
+    .wrap(yargs.terminalWidth())
+    .env("SYNTEST")
```

Comment:
What is the purpose of the env variable?

Reply:
Hmm, not really related to this PR indeed, but I was digging through Yargs during this PR and found that if you specify this Yargs will automatically pick up on environment variables with the prefix "SYNTEST".